jekyll/_cci2/server-3-whats-new.adoc

```diff
@@ -240,7 +240,6 @@ Next, from the KOTS Admin Console, select *Version History* from the menu bar an
 
 === New features
 
-* Telegraf plugins can now be added to server and customized to use third-party monitoring solutions, for example, Datadog. For more information, see the https://circleci.com/docs/server-3-operator-metrics-and-monitoring/[Metrics and Monitoring] doc.
```

Comment:
Is it right to update previous announced features? Were these entries incorrect? Or have these items actually been removed since?

Reply:
@rosieyohannan The issue we have encountered is that while metrics are available, we do not know what to do with them. Customers are often asking how to leverage their metrics and we just don't know. We are now pivoting to just remove metrics and references to them.